requirements.txt

```diff
@@ -0,0 +1,10 @@
+arrow==0.10.0
+boto3
+click==6.7
+click_datetime==0.2
+flake8
+numpy==1.13.3
+pandas
+pyspark==2.2.2
+pytest
```

Comment:
I'd recommend pinning all these dependencies here to aid in repeatability of tests.